.github/workflows/test.yaml

```
@@ -0,0 +1,41 @@
+
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+
+      - name: Test with Gradle
+        run: ./gradlew --info test
+
+      - name: Publish Test Results
+        uses: EnricoMi/publish-unit-test-result-action@v2
+        if: ${{ always() }}
+        with:
+          files: build/test-results/**/*.xml
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)